api_core/google/api_core/protobuf_helpers.py

```diff
@@ -17,7 +17,9 @@
 import collections
 import inspect
 
+from google.protobuf.field_mask_pb2 import FieldMask
 from google.protobuf.message import Message
+from google.protobuf.descriptor import FieldDescriptor
```

Comment:
since you're here, please import modules not members, e.g., change this to:

```
from google.protobuf import descriptor
from google.protobuf import field_mask_pb2
from google.protobuf import message
```

Reply:
Done!